packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx

```diff
@@ -27,7 +31,33 @@ interface Props
  * Handle minimal mode
  * handle tabs
  */
-
+const DashboardOptions = ({
+    isEditMode,
+    data,
+}: {
+    isEditMode: boolean;
+    data: {
+        resultsAndColumns: ResultsAndColumns;
+        chart: SqlChart;
+    };
+}) => {
+    const history = useHistory();
+    return (
+        <Box>
```

Comment:
do we need `Box`?

Reply:
that's what the other options on charts were using, so I added it here too